legacy/src/webhooks/slack.ts

```
@@ -0,0 +1,322 @@
+          const statusIcon = this.getStatusIcon(templateData.status);
+          blocks.push({
+            type: 'context',
+            elements: [
+              { type: 'mrkdwn', text: `${statusIcon} *Status:* ${templateData.status}` }
+            ]
+          });
+        }
+        break;
+
+      case 'progress':
+        let percent = -1;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)